runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/validator/IntegerValidator.java

```diff
@@ -31,8 +31,6 @@ public boolean validate(
         int index,
         int length)
     {
-        byte[] payloadBytes = new byte[length];
-        data.getBytes(0, payloadBytes);
-        return payloadBytes.length == 4 ? true : false;
+        return length == 4 ? true : false;
```

Comment:
No need for ternary operator, since `length == 4` expression is already `boolean`.
So just need `return length == 4;`.
Same feedback for other similar validators.